test/rest/2023_07/customer_address_test.rb

```diff
@@ -46,12 +46,15 @@ def test_1()
     )
 
     assert_requested(:get, "https://test-shop.myshopify.io/admin/api/2023-07/customers/207119551/addresses.json?limit=1")
+    assert_equal(1, response.count)
 
     response = response.first if response.respond_to?(:first)
 
     # Assert attributes are correctly typed preventing Sorbet errors downstream
     if response.respond_to?(:original_state)
       response&.original_state&.each do |key, value|
+        next if key == :default
```

Comment:
I don't like this, but I'm not sure how to better handle. `default` is both an attribute and a method.